.github/workflows/e2e.yml

```diff
@@ -20,6 +20,8 @@ env:
   SIMPLE_REPORT_SENDGRID_ENABLED: false
   SIMPLE_REPORT_SENDGRID_FROM_EMAIL: admin@simplereport.gov
   SIMPLE_REPORT_SENDGRID_ACCOUNT_REQUEST_RECIPIENT: usds@cdc.gov
+  REACT_APP_BASE_URL: http://localhost.simplereport.gov
```

Comment:
Is it necessary to fake a whole domain instead of just using `localhost` here?

Reply:
Yeah, I believe I ran across some documentation somewhere that suggested that a `Set-Cookie` header only works for domains with at least 2 dots in them, ie `localhost.` doesn't work but `localhost.com.` would work.